solutions/level1/book_serializer.rb

```diff
@@ -0,0 +1,13 @@
+require 'ostruct'
+
+class BookSerializer
```

Comment:
why do we need this class ? As I see this creates an OpenStruct similar to Book and calls `.to_h` on it 🤔